website/content/en/preview/tasks/configuration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configuration"
+linkTitle: "Configuration"
+weight: 5
+description: >
+  Configure Karpenter
+---
+
+Karpenter supplies a default configuration that should work for most.  Additional configuration can be performed by 
```

Comment:
Does this configmap exist after Karpenter is deployed, or do you need to create it?